fix(api): dataplane selector control API

timdah opened the pull request:
## What this PR changes/adds

Fixes a dataplane selector control API endpoint.

## Why it does that

Add a missing transformer.

## Further notes

/

## Linked Issue(s)

Closes #4245

ndr-brt reviewed (changes requested):
this could be a good occasion for adding an e2e test on this endpoint

timdah commented:
> this could be a good occasion for adding an e2e test on this endpoint, please fix the pr title and you have to sign the ECA

added an e2e test and fixed the title

ndr-brt reviewed (approved):
LGTM

paullatzelsperger commented on system-tests/e2e-dataplane-tests/tests/src/test/java/org/eclipse/edc/test/e2e/DataplaneSelectorControlApiEndToEndTest.java:
```diff
@@ -0,0 +1,88 @@
+                    "web.http.control.path", "/control",
+                    "edc.dpf.selector.url", String.format("http://localhost:%d/control/v1/dataplanes", controlPlaneControlPort)
+            ),
+            ":core:data-plane:data-plane-core",
```
couldn't you just use `:system-tests:e2e-dataplane-tests:runtimes:data-plane` instead of listing every module separately?

timdah replied:
True! Changed it
